packages/components/organisms/f-checkout/src/components/Header.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+    <div
+        data-test-id="checkoutHeader-component"
+        :class="$style['c-checkoutHeader']">
+        <div
+            v-if="isLoggedIn"
+            data-test-id="user-checkoutHeader">
+            <h2 data-test-id="user-title">
+                {{ $t('checkoutHeader.user.title', { name }) }}
+            </h2>
+
+            <p :class="$style['c-checkoutHeader-loginLink']">
+                <a
+                    :href="loginUrl"
+                    data-test-id="switch-user-link"
+                    @click="onVisitLoginPage">
+                    {{ $t('checkoutHeader.user.switchUser', { name }) }}
+                </a>
+            </p>
+        </div>
+
+        <div
+            v-else
+            data-test-id="guest-checkoutHeader">
+            <h2 :class="$style['c-checkoutHeader-title']">
+                {{ $t('checkoutHeader.guest.guestTitle') }}
+            </h2>
+
+            <a
+                :href="loginUrl"
+                data-test-id="guest-login-button"
+                :class="$style['c-checkoutHeader-loginButton']"
+                @click="onVisitLoginPage">
+                {{ $t('checkoutHeader.guest.loginButton') }}
+            </a>
+
+            <div
+                :class="$style['c-checkoutHeader-option']">
+                <span>{{ $t('checkoutHeader.guest.option') }}</span>
+            </div>
+
+            <h2 :class="$style['c-checkoutHeader-title']">
+                {{ $t('checkoutHeader.guest.guestTitle') }}
+            </h2>
+
+            <p :class="$style['c-checkoutHeader-confirmation']">
+                {{ $t('checkoutHeader.guest.confirmation') }}
+            </p>
+        </div>
+    </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import EventNames from '../event-names';
+
+export default {
+    props: {
+        loginUrl: {
+            type: String,
+            required: true
+        }
+    },
+
+    computed: {
+        ...mapState('checkout', [
+            'customer',
+            'isLoggedIn'
+        ]),
+
+        name () {
+            return (this.customer.firstName.charAt(0).toUpperCase() + this.customer.firstName.slice(1));
+        }
+    },
+
+    methods: {
+        onVisitLoginPage () {
+            this.$emit(EventNames.CheckoutVisitLoginPage);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-checkoutHeader {
+    text-align: center;
+}
+
+.c-checkoutHeader-title {
+    margin-top: 0;
+    margin-bottom: spacing(x0.5);
+    @include font-size('heading-s');
+}
+
+// TODO: Add link button to f-button
```

Comment:
There is a link button so I think this TODO is out of date

Reply:
At the moment there's a button that's styled to look like a link but still works as a button would.
This one needs to be a link that's styled as a button but works as a link.
🤯